coordinator/tests/messagehandling.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {Boom, boomify} from "@hapi/boom"
```

Comment:
Is this file testing any of our code or is it basically testing `boomify`?

Reply:
Good spot. Ill get rid of this